astropy/coordinates/solar_system.py

```diff
@@ -43,79 +43,140 @@
                                        ('neptune', [(0, 8)]),
                                        ('pluto', [(0, 9)]))
                                       )
-SOLAR_SYSTEM_BODIES = tuple(BODY_NAME_TO_KERNEL_SPEC.keys())
 
-_JPL_EPHEM_NOTE = """
-    This calculation uses JPL Ephemeris SPK filesto calculate the body's
-    location, defaulting to DE430.  It will be downloaded the first time this
-    function is used and cached from then on.  To change this, set
-    ``kernel_url`` as described in the coordinate documentation.
+"""Indices to the plan94 routine for the given object."""
+PLAN94_BODY_NAME_TO_PLANET_INDEX = OrderedDict(
+    (('mercury', 1),
+     ('venus', 2),
+     ('earth-moon-barycenter', 3),
+     ('mars', 4),
+     ('jupiter', 5),
+     ('saturn', 6),
+     ('uranus', 7),
+     ('neptune', 8)))
+
+_EPHEMERIS_NOTE = """
+You can either give an explicit ephemeris or use a default, which is normally
+an approximate ephemeris that does not require ephemeris files.  To change
+the default to be a JPL ephemeris::
+
+    >>> from astropy import coordinates as coord
+    >>> coord.solar_system_ephemeris.set('jpl')
+
+This requires the jplephem package (https://pypi.python.org/pypi/jplephem).
+If needed, the ephemeris file will be downloaded (and cached).
+
+One can check which bodies are covered by a given ephemeris using::
+    >>> coord.solar_system_ephemeris.bodies
+    ('earth', 'sun', 'mercury', 'venus', 'earth-moon-barycentre', 'mars',
+     'jupiter', 'saturn', 'uranus', 'neptune')
 """[1:-1]
 
-class kernel_url(ScienceState):
-    """
-    The URL to use for downloading a download the Satellite Planet
-    Kernel (SPK) file with ephemerides.  The download will *not* occur when
-    this state is set, but rather whenever the first time is that the
-    kernel actually needs to be used.
+
+class solar_system_ephemeris(ScienceState):
+    """Default ephemerides for calculating positions of Solar-System bodies.
+
+    This can be a URL to a JPL ephemerides, or one of the following::
```

Comment:
`to a JPL ephemerides`- > `to a JPL binary SPK (bsp) file with the necessary ephemerides`.  (Important to note that it's a bsp file because there are other versions of the SPICE format.)